app/src/main/res/values/strings.xml

```diff
@@ -43,17 +43,17 @@
     <string name="legacy_import_failed">Could not import legacy settings</string>
 
     <string name="auto_updates_dialog_title">Configure updates</string>
-    <string name="auto_updates_dialog_description">Do you want ReVanced Manager to periodically check for updates for the following components?</string>
-    <string name="auto_updates_dialog_manager">ReVanced Manager</string>
-    <string name="auto_updates_dialog_patches">ReVanced Patches</string>
+    <string name="auto_updates_dialog_description">Do you want <nobreak>ReVanced Manager</nobreak> to periodically check for updates for the following components?</string>
+    <string name="auto_updates_dialog_manager"><nobreak>ReVanced Manager</nobreak></string>
+    <string name="auto_updates_dialog_patches"><nobreak>ReVanced Patches</nobreak></string>
```

Comment:
Are you sure `nobreak` works with `stringResource`?

Reply:
I.. forgot

Reply:
It should work if the string is read as an `AnnotatedString`. Compose does not have this currently, but there are snippets online for it.